_includes/v2.0/admin-ui-custom-chart-debug-page-00.html

```diff
@@ -0,0 +1,105 @@
+<table>
```

Comment:
Did you use an all-html table to accommodate bullets? I wonder if it would be easier to use markdown with html in the cells where bullets are needed? cc @lhirata, given that we just wrote these [style guidelines](https://github.com/cockroachdb/docs/blob/master/STYLE.md#tables).

Reply:
Correct, I used HTML because of the bullets.  My attempts to get bulleted lists into Markdown with the pipe characters were ... mostly in vain, I'm afraid.  I did update the HTML with the `thead` and `tbody` tags as you suggested, and they are being formatted like the tables on other pages now.

Re: style guide, I have some feedback/questions in to Lauren via email on just this issue (complex-ish tables).  Definitely want to iron this out so we have consistency, as much as possible!